### PR TITLE
Aegis update + patient clothing

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request
The Aegis:

I've touched up the central hallway's decals, they always bothered me but at the time of the Aegis' creation, I wasn't as well versed with map making as I am now. They're small tweaks that look better to me. The smart fridge was removed and turned into a board, originally you could access the smart fridge from outside of the ship, which wasn't intended. Added some new posters. Compressed the number of red lockers down and cleaned up the main hallway. I gave the psychologist a dart gun because honestly, it sounded really funny.

dart riffle

Also, there are more seeds to make food with. There were like only two food seeds and like five flowers before.

The Patient set:

patient

Kepori and Vox versions are available. Long-term patients now spawn with the white gown and slippers. The previous clothes they spawned with were always intended to be replaced.

Hopefully, this PR also fixes long-term patients spawning with borked ID's. 

## Why It's Good For The Game

Fixing stuff, making stuff look better, and adding new RP opportunities with clothes are good.

## Changelog

:cl: Drag
add: Adds a bunch of shit to the Aegis, nothing earth shattering
add: Added the patient set, along with Vox and Kepori compatible sprites
tweak: tweaked with the Aegis' decals
fix: (Hopefully) Fixes the Aegis' patient Id's
:cl:

### CODE REVIEW

**Review by Imaginos16 — APPROVED** [review]
Good sprites. Cre is cool!

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
I don't think it's the smartest thing to put the weapons in the cryo room. Maybe in a safe on the bridge?
StrongDMM_wRMAGTuroy
Ditch the codespeak, or make it 1 use (captain only)
StrongDMM_3laX9v7Ikj
ditch the guerillas
StrongDMM_dgixa74FQe
you can fit the fridge here, you just need to put it on a tile adjacent to the wall, pixelshift it, and set it's density to zero
StrongDMM_RD4NvA5Qky
Give this door a button on the interior (for the psychologist) that has the special functions var set to three. It will bolt the door, and allow for some privacy. The door's id_tag var will need to match the button's id var.
The bridge could use a bolt button too.
image
cute but you can force pacifism on someone with this. If you want it _because_ it's cute, you need to make a safe subtype.
StrongDMM_xZvIB4w8TV
don't run this wire under the SSU
StrongDMM_yDfgkwZmHR
given that the ship's focus isn't botany. Biogenerator *or* nutrivend.
StrongDMM_Ri9swx64uO
adjust the layering on this railing so it doesn't look like this.
StrongDMM_TUzH2f88KC
the catwalk will go over this decal, making this tile look stupidly busy
StrongDMM_S4DeT50Olq
use the /ship variant of grass

**Comment by Draggeru** [comment]
>[StrongDMM_wRMAGTuroy](https://user-images.githubusercontent.com/94164348/252559931-010d7bf2-0d3f-42bb-90fc-bbeabd8d4fd2.png) Ditch the codespeak, or make it 1 use (captain only)

The codespeak manual has been available and approved from day one, is there a reason why we no longer want the entire crew with codespeak?

**Comment by Erikafox** [comment]
> The codespeak manual has been available and approved from day one, is there a reason why we no longer want the entire crew with codespeak?

for clarity - 
Discord_hpcj4Ne8md

> Give this door a button on the interior (for the psychologist) that has the special functions var set to three. It will bolt the door, and allow for some privacy. The door's id_tag var will need to match the button's id var.
The bridge could use a bolt button too.

I was informed that this is actually function 4, and requires the normaldoorfunction var to be set to 1

**Comment by Draggeru** [comment]
Everything but the grass was changed, applying the ship version of the grass breaks.

**Review by Erikafox — CHANGES_REQUESTED** [review]
You've got some tables that are just. greyscaled to be green in botany

And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.

**Comment by Draggeru** [comment]
> You've got some tables that are just. greyscaled to be green in botany
> 
> And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.

Is there an issue with the tables being green? There are tables that are greyscaled orange in engineering, why is there no apparent problem with them?

**Comment by Erikafox** [comment]
> > You've got some tables that are just. greyscaled to be green in botany
> > And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.
> 
> Is there an issue with the tables being green? There are tables that are greyscaled orange in engineering, why is there no apparent problem with them?

I thought they were wooden tables.

**Comment by Draggeru** [comment]
> > > You've got some tables that are just. greyscaled to be green in botany
> > > And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.
> > 
> > 
> > Is there an issue with the tables being green? There are tables that are greyscaled orange in engineering, why is there no apparent problem with them?
> 
> I thought they were wooden tables.

So just change all the tabled to wooden tables and its good to go?

**Comment by Erikafox** [comment]
> > > > You've got some tables that are just. greyscaled to be green in botany
> > > > And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.
> > > 
> > > 
> > > Is there an issue with the tables being green? There are tables that are greyscaled orange in engineering, why is there no apparent problem with them?
> > 
> > 
> > I thought they were wooden tables.
> 
> So just change all the tabled to wooden tables and its good to go?

Ungreen the green tables and swap the back tables out for wood ones I suppose.

**Comment by Draggeru** [comment]
> And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.

Are we going forward with limiting all ships to one at some point since we are seeing active new ship projects? (Including the Eventide)

**Comment by Erikafox** [comment]
> > And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.
> 
> Are we going forward with limiting all ships to one at some point since we are seeing active new ship projects? (Including the Eventide)

The default is going to become 1 Soonish.

**Comment by Draggeru** [comment]
> > > And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.
> > 
> > 
> > Are we going forward with limiting all ships to one at some point since we are seeing active new ship projects? (Including the Eventide)
> 
> The default is going to become 1 Soonish.

While I have your attention, should the hardline logo be swapped out with the ISF logo? Hardline makes industrial ships and the Aegis is not intended as one. What is the determining factor for manufacturing logos?

**Comment by Erikafox** [comment]
> > > > And I think this should be config limited to 1 ship per round. It can take up a good amount of crew.
> > > 
> > > 
> > > Are we going forward with limiting all ships to one at some point since we are seeing active new ship projects? (Including the Eventide)
> > 
> > 
> > The default is going to become 1 Soonish.
> 
> While I have your attention, should the hardline logo be swapped out with the ISF logo? Hardline makes industrial ships and the Aegis is not intended as one. What is the determining factor for manufacturing logos?

I would ask the lore team

**Comment by Draggeru** [comment]
What Var should be added to the json to limit the Aegis to one ship, Im officially having a certified Drag moment
Edit: Nvm I found it.

**Comment by Draggeru** [comment]
This is where we are at so far appearance wise

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -182,3 +182,31 @@
...
+/obj/item/clothing/under/misc/gown //Clothing for medical ships
+	name = "patient gown"
+	desc = "A light white gown that allows easy access to any patient who wears this."
+	icon = 'icons/obj/clothing/patient.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/patient.dmi'
+	lefthand_file = 'icons/mob/inhands/patient_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/patient_righthand.dmi'
+	icon_state = "gownwhite"
+	item_state = "gownwhite"
+	supports_variations = VOX_VARIATION | KEPORI_VARIATION
```
Comment (thgvr) [inline code/modules/clothing/under/miscellaneous.dm]:
```suggestion
	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON | VOX_VARIATION | KEPORI_VARIATION
```

**Review by Apogee-dev — DISMISSED** [review]
Add access restrictions, in general. Workspaces and critical areas like engineering, the medical labs, the kitchen, and ESPECIALLY the bridge should have appropriate access restrictions on their doors.

**Comment by Draggeru** [comment]
> Add access restrictions, in general. Workspaces and critical areas like engineering, the medical labs, the kitchen, and ESPECIALLY the bridge should have appropriate access restrictions on their doors.

Ill put access restrictions on engineering, but the bridge has a door lock, medical should not have ANY restrictions due to its nature with the long term patient (and the fact you must walk through medical to get to the janitorial supplies), the kitchen and botany are intended as public access. The bridge has no windows(that would allow the crew to speak to anyone in the cockpit), and there have been no issues with people invading the space in my playtesting on the live server. If it is a problem there is a lock that can be applied.

**Review by Erikafox — APPROVED** [review]

**Pull request merged**